include/dpp/channel.h

```diff
@@ -123,6 +147,15 @@ class channel : public managed {
 	/** Permission overwrites to apply to base permissions */
 	std::vector<permission_overwrite> permission_overwrites;
 
+	/** Approximate count of messages in a thread (threads) */
+	uint8_t message_count;
+
+	/** Approximate count of members in a thread (threads) */
```

Comment:
Remember to init both of these to 0 in channel::channel

Reply:
resolved